apidoc/Titanium/Media/Media.yml

```diff
@@ -1265,17 +1277,30 @@ properties:
     type: Number
     permission: read-only
     platforms: [android, iphone, ipad]
+  - name: VIDEO_LOAD_STATE_FAILED
+    summary: Indicates that the player can no longer play AVPlayerItem instances because of an error.
```

Comment:
Titanium users have no idea what a `AVPlayerItem` is. Use "media items" instead.